src/components/KFileUpload/KFileUpload.vue

```diff
@@ -26,9 +29,9 @@
       </span>
 
       <KInput
+        :id="inputId"
```

Comment:
> why aren’t we using v-bind-once directive here?

As I mentioned in the PR description, using `v-bind-once` on custom components binds the attributes to the root node of the custom component, regardless of its `inheritAttrs` configuration. If we add `v-bind-once="{ id: inputId }"` here, the root node (`div.k-input`) of `KInput` will have an id of `inputId` as well, even though we set `inheritAttrs: false` in `KInput`.

> we still need to use the directive to ensure SSR compatibility

Adding `:id="inputId"` on `KInput` should be SSR safe because a) `KInput` is a custom component, and b) `KInput` passes the id to its `<input>` element using `v-bind-once`: https://github.com/Kong/kongponents/blob/5f9232c61bd72d7519c4c653d346e9372c926195/src/components/KInput/KInput.vue#L35.